CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Things you will need
 
  * Linux or Mac OS X. (Windows is not yet supported.)
  * git (used for source version control).
- * An IDE. We recommend [IntelliJ with the Flutter plugin](https://flutter.io/intellij-ide/).
+ * An IDE. We recommend [IntelliJ with the Flutter plugin](https://flutter.io/intellij-ide/) or Xcode.
```

Comment:
IntelliJ and Xcode are in different categories.  IntelliJ is what we recommend for editing the Dart code in the flutter.git repository.  Xcode is a tool that's required in order to build apps for iOS.